fdbserver/workloads/ConsistencyCheckUrgent.actor.cpp

```diff
@@ -0,0 +1,605 @@
+/*
+ * ConsistencyCheckUrgent.actor.cpp
+ *
+ * This source file is part of the FoundationDB open source project
+ *
+ * Copyright 2013-2022 Apple Inc. and the FoundationDB project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <math.h>
+#include "boost/lexical_cast.hpp"
+
+#include "flow/IRandom.h"
+#include "flow/ProcessEvents.h"
+#include "fdbclient/NativeAPI.actor.h"
+#include "fdbclient/FDBTypes.h"
+#include "fdbserver/TesterInterface.actor.h"
+#include "fdbserver/workloads/workloads.actor.h"
+#include "flow/IRateControl.h"
+#include "fdbrpc/simulator.h"
+#include "fdbserver/Knobs.h"
+#include "flow/DeterministicRandom.h"
+#include "fdbclient/ManagementAPI.actor.h"
+#include "fdbclient/StorageServerInterface.h"
+#include "flow/network.h"
+#include "fdbrpc/SimulatorProcessInfo.h"
+#include "flow/actorcompiler.h" // This must be the last #include.
+
+struct ConsistencyCheckUrgentWorkload : TestWorkload {
```

Comment:
Maybe add some comments about this workload. How is it different from existing consistency checker.